metaworld/envs/mujoco/sawyer_xyz/v2/sawyer_faucet_close_v2.py

```diff
@@ -43,36 +44,32 @@ def model_name(self):
 
     @_assert_task_is_set
     def evaluate_state(self, obs, action):
-        reward, reachDist, pullDist = self.compute_reward(action, obs)
+        (reward, tcp_to_obj, _, target_to_obj, object_grasped,
+         in_place) = self.compute_reward(action, obs)
+
         info = {
-            'reachDist': reachDist,
-            'goalDist': pullDist,
-            'epRew': reward,
-            'pickRew': None,
-            'success': float(pullDist <= 0.05),
+            'success': float(target_to_obj <= 0.11),
+            'near_object': float(tcp_to_obj <= 0.01),
+            'grasp_success': 1.,
+            'grasp_reward': object_grasped,
+            'in_place_reward': in_place,
+            'obj_to_target': target_to_obj,
+            'unscaled_reward': reward,
         }
 
         return reward, info
 
     @property
     def _target_site_config(self):
-        return [
-            ('goal_close', self._target_pos),
-            ('goal_open', np.array([10., 10., 10.]))
-        ]
-
-    def _get_pos_objects(self):
-        knob_center = self.get_body_com('faucetBase') + np.array([.0, .0, .125])
-        knob_angle_rad = self.data.get_joint_qpos('knob_Joint_1')
+        return [('goal_close', self._target_pos),
+                ('goal_open', np.array([10., 10., 10.]))]
 
-        offset = np.array([
-            np.sin(knob_angle_rad),
-            -np.cos(knob_angle_rad),
-            0
-        ])
-        offset *= self.HANDLE_LENGTH
+    def _get_quat_objects(self):
+        return self.sim.data.get_body_xquat('faucetBase')
 
-        return knob_center + offset
+    def _get_pos_objects(self):
+        return self._get_site_pos('handleStartClose') + np.array(
```

Comment:
Why did you change my code here? It returns the position of the end of the faucet at all times. What you have here just returns the position of a static site

Reply:
@hayden the sites before hand were actually not moving with the faucet arm. 

I changed it so that they do, so now the site is always sticking to the faucet.